setup.cfg

```diff
@@ -20,6 +20,7 @@ install_requires =
     pandas~=1.3
```

Comment:
I'm curious why the install_requires are set to "~=" instead of >= or with upper/lower bounds like I added for pyarrow.  I couldn't find other examples using "~=" online, what is the purpose of that approach?

Reply:
@gcnuss The pattern is aligned to [this](https://peps.python.org/pep-0440/#compatible-release) reference.